docs/dev/run-cvo-locally.md

```diff
@@ -0,0 +1,51 @@
+# How to Build and Run CVO Locally
+
+This document provides the instructions to build and run the CVO executable locally. Local execution of CVO, for example executing CVO on your development laptop, allows a "quick-turn" test capability whereby the CVO executable can be quickly rebuilt with changes and launched again without the need of starting a new OCP cluster.
+
+## Preconditions
+
+Build the CVO executable using the standard procedure:
+
+```console
+$ cd ~/go/src/github.com/openshift/cluster-version-operator
+$ make
+```
+
+Start an OCP cluster of the appropriate version. One that is compatiable with the version of CVO with which you're working.
+
+Extract and download the OCP cluster release:
+
+```console
+$ oc image extract -a /home/jottofar/pull-secret-internal quay.io/openshift-release-dev/ocp-release:4.4.0-rc.4-x86_64 --path /:/home/jottofar/tmp/release/
```

Comment:
[More][1] `/home/jottofar/` prefixes here and later in this doc.

[1]: https://github.com/openshift/cluster-version-operator/pull/381#discussion_r436867790

Reply:
Sorry! Fixed in [20e42a3](https://github.com/openshift/cluster-version-operator/pull/381/commits/20e42a306c3db2d1c487c7dd266ed92cc1e3f024)